packages/node/test/transports/http.test.ts

```diff
@@ -27,11 +28,9 @@ function createTransport(options: TransportOptions): HTTPTransport {
 }
 
 function assertBasicOptions(options: any): void {
-  expect(options.headers['X-Sentry-Auth']).toContain('sentry_version');
-  expect(options.headers['X-Sentry-Auth']).toContain('sentry_client');
-  expect(options.headers['X-Sentry-Auth']).toContain('sentry_key');
+  expect(options.headers).not.toContain('X-Sentry-Auth'); // auth is part of the query string
```

Comment:
Please note this change. In order to reuse code between browser and node, we always send auth as part of the query string (and never send `sentry_client` anymore -- full SDK info is in the event payload).